projects/bms_carrier/src/fan_control.c

```diff
@@ -0,0 +1,50 @@
+#include "bms.h"
+#include "math.h"
+
+static Adt7476aStorage adt7476a_storage;
+static SoftTimerId s_timer_id;
```

Comment:
I don't think you use this, can probably remove it

Reply:
instead of passing `s_timer_id` in `soft_timer_start_millis`, should it be replaced with something like `SOFT_TIMER_INVALID_TIMER` there too?

Reply:
You can pass `NULL` there, generally you only need to pass a non-null pointer there if you're going to cancel the timer later.